ideasbox/templates/index.html

```diff
@@ -3,17 +3,18 @@
 {% load i18n ideasbox_tags %}
 
 {% block content %}
-    <div class="row">
-        <div class="col half">
-            <h2 class="flow"><span class="theme create">blog</span><span>{% trans "LAST POSTS" %}</span></h2>
-            {% for content in blog_contents %}
-                {% include 'blog/content_card.html' %}
-            {% endfor %}
-        </div>
-        <div class="col half">
-            {% include 'search/box.html' %}
+    <div class="row column_small_screen">
```

Comment:
I see. Just noticed I never added any `@media` rules in the CSS, they must lie down in some branch :p
Anyway, to keep it consistent with the naming, what I usually do is add classes like `mwide`, `twide`, `mquarter`, `tquarter` which means "wide on **m**obile", "wide on **t**ablet", etc. So if I understand correctly what you are achieving, this would be `mwide`. See https://github.com/etalab/adresse.data.gouv.fr/blob/master/adresse/static/main.css#L1255-L1321 for an example.
Ping me on IRC if it's not clear :)